StyleCop.Analyzers/StyleCopTester/Program.cs

```diff
@@ -39,6 +39,10 @@ private static void Main(string[] args)
                     cts.Cancel();
                 };
 
+            // The following allows continuations to be processed by a single thread.
```

Comment:
:bulb: I think a good approach would be:

1. Remove the first line of this comment.
2. Add a second informational link to https://github.com/DotNetAnalyzers/StyleCopAnalyzers/pull/1362.